.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,141 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+  "name": "Python 3",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  // TODO: Bullseye has glibc 2.31 and cadquery-ocp@7.7.2 needs 2.35
+  "image": "mcr.microsoft.com/devcontainers/python:1-3.10-bookworm",
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "ghcr.io/deep-space-cartel/devcontainers-features/apt:latest": {
+      // TODO: cmake is a dependency of Python package "pycairo"
+      // TODO: as well as those: libcairo2-dev pkg-config python3-dev // https://pycairo.readthedocs.io/en/latest/getting_started.html
+      // ffmpeg libsm6 libxext6 to fix ImportError: libGL.so.1: cannot open shared object file: No such file or directory
+      // openscad: is needed for "pc inspect :test"
+      "packages": "shellcheck bash-completion cmake libcairo2-dev pkg-config python3-dev ffmpeg libsm6 libxext6 openscad" // shfmt
+    },
+    "ghcr.io/deep-space-cartel/devcontainers-features/pip:1": {
+      "packages": "pre-commit poetry",
+      "location": "/opt/pip"
+    },
+    "ghcr.io/deep-space-cartel/devcontainers-features/starship:latest": {
+      "version": "latest"
+    }
+  },
+  "remoteEnv": {
+    "PATH": "${containerEnv:PATH}:/opt/pip/bin"
+  },
+  // Use 'forwardPorts' to make a list of ports inside the container available locally.
+  // "forwardPorts": [],
+  // Use 'postCreateCommand' to run commands after the container is created.
+  // "postCreateCommand": "pip3 install --user -r requirements.txt",
+  // Configure tool-specific properties.
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "launch": {
+          "version": "0.2.0",
+          "configurations": [
+            {
+              "name": "Python: Current File",
+              "type": "debugpy",
+              "request": "launch",
+              "program": "${file}",
+              "console": "integratedTerminal",
+              "justMyCode": true
+            }
+          ]
+        },
```

Comment:
It would be nice to add debugger configuration for `poetry` since that glibc dependency was not obvious, as well as configuration to run `pc` with debugger.